<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianBlurData</class>
 <widget class="QWidget" name="GaussianBlurData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gaussian Blur constants</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="GaussianBlurX">
     <property name="title">
      <string>X Size</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="xSizeBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GaussianBlurY">
     <property name="title">
      <string>Y Size</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="ySizeBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GaussianBlurSigma">
     <property name="title">
      <string>Sigma</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QDoubleSpinBox" name="sigmaBox">
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okPushButon">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
